lib/board.js

```diff
@@ -51,50 +66,80 @@ function Board(options){
 
 util.inherits(Board, EventEmitter);
 
-Board.search = function(portList, options, cb){
-  var revisions = _.keys(bs2.revisions);
-  return bluebird.reduce(portList, function(boardList, path){
+Board.listPorts = function listPorts(options){
```

Comment:
don't expose this on board (it is only used internally and only supports promises